bin/scotty.js

```diff
@@ -72,14 +73,18 @@ function readArgs() {
       s: 'source',
       b: 'bucket',
       r: 'region',
+      d: 'domain',
+      z: 'zoneId',
       f: 'force',
       u: 'update'
     },
-    string: ['source', 'bucket', 'region'],
+    string: ['source', 'bucket', 'region', 'domain', 'zoneId'],
     boolean: ['quiet', 'website', 'spa', 'force', 'update'],
     default: {
       source: process.cwd(),
-      bucket: path.parse(process.cwd()).name
+      bucket: path.parse(process.cwd()).name,
+      zone: false,
+      name: false
```

Comment:
No need to set false for `name` here. It will be undefined by default, so `if` works as expected.